### PR TITLE
Visiony rebase

### PR DESCRIPTION
**Pull request by superyuuki**

### CODE REVIEW

**Comment by superyuuki** [comment]
@cpostbitbuckets @arcaputo  needs review

File: src/main/java/org/bitbuckets/robot/RobotSetup.java [hunk 1]
```diff
@@ -45,7 +44,7 @@ public RobotContainer build(ProcessPath path) {
         }
 
 
-        return new RobotContainer(driveSubsystem, armSubsystem, elevatorSubsystem);
+        return new RobotContainer(driveSubsystem, armSubsystem, elevatorSubsystem, visionControl);
```
Comment (cpostbitbuckets) [inline src/main/java/org/bitbuckets/robot/RobotSetup.java:47]: VisionControl not be on the same level as a subsystem, but we don't have to fix that on this branch.

File: src/main/java/org/bitbuckets/robot/RobotContainer.java [hunk 2]
```diff
@@ -27,13 +33,17 @@ public void robotPeriodic() {
         driveSubsystem.robotPeriodic();
     }
 
+
+    int i = 0;
```
Comment (cpostbitbuckets) [inline src/main/java/org/bitbuckets/robot/RobotContainer.java:37]: seems like it shouldn't be here.

**Review by cpostbitbuckets — CHANGES_REQUESTED** [review]

File: src/main/java/org/bitbuckets/drive/controlsds/DriveControl.java [hunk 3]
```diff
@@ -65,8 +65,8 @@ public DriveControl(ISwerveModule moduleFrontLeft, ISwerveModule moduleFrontRigh
     public void guaranteedLoggingLoop() {
         Logger.getInstance().recordOutput("a", reportSetpointStates());
 
-        desiredStates.log(reportSetpointStates());
-        actualStates.log(reportActualStates());
+        //desiredStates.log(reportSetpointStates());
+        //actualStates.log(reportActualStates());
```
[the remark this answers is not in the document]
Comment (superyuuki) [reply src/main/java/org/bitbuckets/drive/controlsds/DriveControl.java:69]: it shouldn't be

File: src/main/java/org/bitbuckets/bootstrap/Robot.java [hunk 4]
```diff
@@ -46,11 +46,11 @@ public void robotInit() {
 
         if (isReal()) {
             logger.addDataReceiver(new WPILOGWriter("/media/sda1/")); // Log to a USB stick
-            logger.addDataReceiver(new NetworkPublisher()); // Publish data to NetworkTables
```
[the remark this answers is not in the document]
Comment (superyuuki) [reply src/main/java/org/bitbuckets/bootstrap/Robot.java:49]: this shouldn't be happening either

File: src/main/java/org/bitbuckets/robot/RobotContainer.java (hunk 2 above)
Comment (superyuuki) [reply src/main/java/org/bitbuckets/robot/RobotContainer.java:37]: seems like a debug thing

File: src/main/java/org/bitbuckets/robot/RobotSetup.java (hunk 1 above)
Comment (superyuuki) [reply src/main/java/org/bitbuckets/robot/RobotSetup.java:47]: we found it!

Comment (cpostbitbuckets) [reply src/main/java/org/bitbuckets/robot/RobotSetup.java:47]: ?

**Pull request closed without merging**